data-serving/scripts/README.md

```diff
@@ -14,37 +14,92 @@ Logs will be written to `convert_data.log`.
 
 ### Current stats
 
-- 99.7% of rows from the CSV file convert successfully to JSON
-  + Of the 0.3% of rows with errors:
-    - 99.6% have age values outside the valid range
-    - 0.4% have invalid date formats
-- 100% of rows from `cases.json` import successfully into mongodb using `cases.schema.json` as a validator
+- 99.2% of rows from the CSV file convert entirely successfully to JSON. For those with errors, only the failed fields are ommitted.
+- 100% of rows from `cases.json` validate and import successfully into mongodb.
+
+### Lossy fields
+
+A few fields can't be converted losslessly because they need to be normalized upon insertion into the new data format. All of the original values are strings; some of the target representations are bools, ints, floats, or enums. Additionally, the new schema has validation rules and invalid values, e.g. an age over 300. When a value can't be successfully converted, the failure is logged in `convert_data.log` and the value for that field is dropped (though again, the original value is archived in the approrpiate `originalCase` field).
```

Comment:
Mind linebreaking this file? 100 or 120 if you can deal with context switching from code

Reply:
sure, did 120